internal/impl/devbox.go

```diff
@@ -437,21 +536,20 @@ func (d *Devbox) StartProcessManager(
 		}
 	}
 	if !IsDevboxShellEnabled() {
-		args := []string{"services", "manager"}
+		args := []string{"services", "up"}
+		args = append(args, requestedServices...)
 		if processComposeFileOrDir != "" {
 			args = append(args, "--process-compose-file", processComposeFileOrDir)
 		}
+		if background {
+			args = append(args, "--background")
+		}
 		return d.RunScript("devbox", args)
 	}
 
-	return services.StartProcessManager(ctx, processComposePath, svcs, processCompose)
-}
+	// Start the process manager
 
-func (d *Devbox) StopServices(ctx context.Context, serviceNames ...string) error {
-	if !IsDevboxShellEnabled() {
-		return d.RunScript("devbox", append([]string{"services", "stop"}, serviceNames...))
-	}
-	return services.Stop(ctx, d.mergedPackages(), serviceNames, d.projectDir, d.writer)
+	return services.StartProcessManager(ctx, requestedServices, svcs, d.projectDir, processComposePath, processComposeFileOrDir, background)
```

Comment:
@Lagoja one problem is that the `processComposePath` here is empty if line 531-532 returned an error of "not found".

After we install process-compoe in lines 533-535, we need re-run `processComposePath, err := utilityLookPath("process-compose")`

Reply:
@savil good catch! 

Should we modify `addDevboxUtilityPackage` to return the path? (e.g. `(path, err)`)

Reply:
oh I just saw this. I made a commit to just call `utilityLookPath` again, and see if that resolves the other cli-test issues in buildkite.

re: moving it inside addDevboxUtilityPackage. I feel its simpler if that just does the "adding" and the caller can decide if they need to re-lookup. Other callers may not have a need for the package's path...

Reply:
...and buildkite run is Green :yay: